app/routes/home.tsx

```
@@ -68,32 +69,81 @@ export let handle: RouteHandle = {
+						</div>
+
+						{/* Social Links */}
+						<div className="mt-8 flex gap-4 justify-center md:justify-start">
+							<a
+								href="https://github.com/mcansh"
+								target="_blank"
+								rel="noopener noreferrer"
+								className="inline-flex items-center gap-2 px-6 py-3 bg-gray-900 text-white rounded-lg hover:bg-gray-800 transition-colors shadow-lg hover:shadow-xl"
+							>
+								<svg className="size-5" aria-hidden>
+									<use href={lucideGithubIconHref} />
```

Biome (2.1.2)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)